Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:buster-slim
```

Comment:
Also, is this the equivalent of using the latest version of image `debian-buster-slim`? If it is, then we should consider pinning it to an LTS version, to avoid any surprises when/if they update the image with breaking changes.

Reply:
No, this is not quite the same. Buster is the LTS version

Reply:
Perfect!